armory/art_experimental/attacks/kenansville_dft.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+
+
+class KenansvilleDFT:
+    def __init__(
+        self,
+        estimator,
+        sample_rate=16000,
+        snr_db=100,
+        partial_attack=False,
+        attack_len=500,
+        attack_prob=0.5,
+        targeted=False,
```

Comment:
So that it is more clear to the user that `targeted` is not used, we should add an assert statement to verify it is False.